Makefile

```diff
@@ -142,7 +142,7 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 	$(KUSTOMIZE) build config/crd | kubectl apply -f -
 
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
+	$(KUSTOMIZE) build config/crd | kubectl delete --ignore-not-found=$(ignore-not-found)  -f -
```

Comment:
nit: just a difference of a space?

Reply:
this needs change, that I will be doing it in separate PR later. for now marcus removed the change , i think he left a space.

Reply:
done in the latest commit.